templates/yeet/src/probe.bpf.c

```diff
@@ -0,0 +1,27 @@
+#include <vmlinux.h>
+#include <bpf/bpf_helpers.h>
+#include <yeet/yeet.h>
+
+struct output {
+  int pid;
+} __attribute__((packed));
+
+RINGBUF_CHANNEL(output_rb, 1024 * sizeof(struct output), output);
+
+SEC("tracepoint/syscalls/sys_enter_execve")
```

Comment:
should we make the example something that is semi useful like how react has the initial page.